be/src/exprs/bloom_filter_func.h

```diff
@@ -341,6 +379,44 @@ struct DateFindOp : public CommonFindOp<DateTimeValue> {
     void insert(BloomFilterAdaptor& bloom_filter, const void* data) const {
         bloom_filter.add_bytes((char*)data, sizeof(vectorized::VecDateTimeValue));
     }
+
```

Comment:
why directly use parent class `CommonFindOp<VecDateTimeValue>` not we do not have old column exec engine.